doc/man1/openssl-req.pod.in

```diff
@@ -652,6 +660,10 @@ Examine and verify certificate request:
 
  openssl req -in req.pem -text -verify -noout
 
+Specify the cipher to be used for encrypting the private key:
+
+ openssl req -new -key privatekey.pem -out request.csr -cipher aes256
```

Comment:
This example isn't quite right. "-new" only means to create a new CSR, not a new key. The "-key" option specifies the input key file. So this doesn't actually generate a new key at all. Also "aes256" here is just an infrequently used alias for "aes-256-cbc" - so it would be better to use that instead.

So, a better example might be:

````
openssl req -newkey rsa:2048 -keyout privatekey.pem -out request.csr -cipher aes-256-cbc
````